.rubocop.yml

```diff
@@ -4,9 +4,18 @@ require:
 AllCops:
     NewCops: enable
 
+Metrics/AbcSize:
+  Exclude:
+    - 'db/migrate/*'
+
 Metrics/BlockLength:
   Exclude:
     - 'lib/tasks/auto_annotate_models.rake'
+    - 'spec/requests/registrations_spec.rb'
```

Comment:
Can we fix this cop instead of ignoring it?

Reply:
I would disable this feature for specs since these usually spends a large blocks of code when we validate multiple scenarios and response values.